Doctrine/Phpcr/MediaManager.php

```diff
@@ -133,8 +134,12 @@ public function setDefaults(MediaInterface $media, $parentPath = null)
      */
     public function mapPathToId($path, $rootPath = null)
     {
-        // The path is being the id
-        $id = PathHelper::absolutizePath($path, '/');
+        try {
+            // The path is being the id
+            $id = PathHelper::absolutizePath($path, '/');
+        } catch (RepositoryException $e) {
+            throw new \OutOfBoundsException($e->getMessage(), 0, $e);
```

Comment:
yes, lets declare those exceptions on this method and catch them both in the controller and throw a NotFoundHttpException with the exception as root cause.